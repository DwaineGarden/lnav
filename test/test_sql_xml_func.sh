#! /bin/bash

run_cap_test ./drive_sql "SELECT * FROM xpath('/abc[', '<abc/>')"

run_cap_test ./drive_sql "SELECT * FROM xpath('/abc', '<abc')"

run_cap_test ./drive_sql "SELECT * FROM xpath('/abc/def', '<abc/>')"

run_cap_test ./drive_sql "SELECT * FROM xpath('/abc/def[@a=\"b\"]', '<abc><def/><def a=\"b\">ghi</def></abc>')"

run_cap_test ./drive_sql "SELECT * FROM xpath('/abc/def', '<abc><def>Hello &gt;</def></abc>')"
