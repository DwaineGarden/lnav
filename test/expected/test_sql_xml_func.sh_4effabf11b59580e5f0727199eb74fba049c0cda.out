Row 0:
  Column     result: <def>Hello &gt;</def>

  Column  node_path: /abc/def
  Column  node_attr: {}
  Column  node_text: Hello >
